src/lib/util/validateAll.js

```diff
@@ -0,0 +1,21 @@
+
+import assertString from './assertString';
+
+export default function validateAll(value, ...validatorFunctions) {
+  assertString(value);
+  const isValidatorFunctionsExist = validatorFunctions.length;
+
+  if (!isValidatorFunctionsExist) {
+    throw new Error("validator functions can'nt be empty");
+  }
+
+  for (let i = 0; i < validatorFunctions.length; i++) {
+    const currentValidatorFunction = validatorFunctions[i];
+    const isValid = currentValidatorFunction(value);
+    if (!isValid) {
+      return false;
+    }
+  }
```

Comment:
Maybe we should return an array with all the results instead of exiting directly when a value is invalid?

Reply:
Thanks form quick response. 

I think if we write function in this way then users must reduce array to know the result of validateAll function.
and that's pointless.